doc/nrf/create_application.rst

```diff
@@ -0,0 +1,216 @@
+.. _create_application:
+
+Create an application
+#####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The process for creating an application in the |NCS| follows Zephyr's :ref:`Application Development principles <zephyr:application>` and the |NCS|'s :ref:`dev-model`.
+The following sections provide basic information based on these two sources.
+
+.. _create_application_structure:
+
+Application structure
+*********************
+
+Here are the files that most of the Zephyr and |NCS| applications include in its :file:`<app>` directory:
+
+.. code-block:: none
+
+   <app>
+   ├── CMakeLists.txt
+   ├── app.overlay
+   ├── prj.conf
+   └── src
+       └── main.c
+
+You can read more about each of these files in Zephyr's :ref:`Application Development overview <zephyr:application>`.
+
+.. _create_application_types:
+
+Application types
+*****************
+
+Based on where the :file:`<app>` directory is located, you will usually work with one of two types of applications in the |NCS|: workspace or freestanding.
+These types are adopted from :ref:`Zephyr application types <zephyr:zephyr-app-types>`.
+
+.. _create_application_types_workspace:
+
+Workspace application
+=====================
+
+A workspace application is located inside a west workspace, but outside of the SDK repository.
```

Comment:
To be precise the SDK is not a single repository but a collection of repositories.